files/ptn0/scripts/prereqs.sh

```diff
@@ -58,9 +58,9 @@ echo "Installing Cabal 3.0.0 .."
 wget https://downloads.haskell.org/cabal/cabal-install-3.0.0.0/cabal-install-3.0.0.0-x86_64-unknown-linux.tar.xz
 tar xf cabal-install-3.0.0.0-x86_64-unknown-linux.tar.xz
 chmod 755 cabal
-mkdir -p ~/.cabal/bin
```

Comment:
~/.cabal/bin is used as destination for output of cabal build outputs (part of cabal-built-all.sh), which resembles standard behaviour for cabal install mode

Reply:
Than we also need it to be included in the PATH as well:
Warning: Installation path /root/.cabal/bin not found on the PATH environment variable.

Reply:
Yep which is done by ghcup install too (atleast in interactive mode)